.gitignore

```diff
@@ -0,0 +1,55 @@
+# Compiled documentation
+propagation/SWALS/doc
+
+# Test outputs
+propagation/SWALS/tests/*/OUTPUTS
+propagation/SWALS/tests/*/*.log
+propagation/SWALS/tests/*/*unit_tests
+propagation/SWALS/tests/unit_tests/*.csv
+propagation/SWALS/tests/unit_tests/*.nc
+
+# Example outputs
+propagation/SWALS/examples/**/OUTPUTS
+propagation/SWALS/examples/**/*.png
+propagation/SWALS/examples/**/*.log
+propagation/SWALS/examples/**/*.csv
+# Trick to ignore all (binary) files without an extension
+propagation/SWALS/examples/*
```

Comment:
Just double checking that this only ignores binary files without an extension? In other contexts the notation would suggest that all files match.

Reply:
Yes, together lines 17-19 ignores any file without an extension inside the propagation/SWALS/examples directory.